Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/Helpers/GherkinTableNodeBasedContentDimensionSource.php

```diff
@@ -28,6 +28,7 @@
 
 /**
  * The node creation trait for behavioral tests
+ * todo colocate with GherkinTableNodeBasedContentDimensionSourceFactory
```

Comment:
that seems left alone here

Reply:
will be adjusted via https://github.com/neos/neos-development-collection/pull/5346